hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/throttles/GasLimitBucketThrottle.java

```diff
@@ -109,7 +109,7 @@ void reclaimLastAllowedUse() {
     /**
      * Returns an instance of the {@link DiscreteLeakyBucket} used under the hood.
      *
-     * @return an instance of the {@link DiscreteLeakyBucket} used under the hood.
+     * @return an instance of the {@link DiscreteLeakyBucket} used under the hood
```

Comment:
I'm confused why no period is wanted here, but it is wanted on other sentence fragments

Reply:
Please see https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment